Templates/Swift/Pay/Sources/API.swift

```diff
@@ -0,0 +1,55 @@
+{% include "Includes/Header.stencil" %}
```

Comment:
This file is also almost identical to the one in `Poi`, except for the server name. Can we combine these, e.g. in a stencil file?